tests/testthat/test-proxy.R

```diff
@@ -6,7 +6,8 @@ test_that("eval_*() respects proxy settings", {
     },
     tidyselect_data_supports_predicates.foo = function(x) {
       FALSE
-    }
+    },
+    .env = globalenv()
```

Comment:
Oh sorry, I guess I thought this was `local_methods()`

https://github.com/r-lib/vctrs/blob/e86bb9a4b57459eb56099c0b52af37877a9bf515/tests/testthat/helper-s3.R#L34-L36